cmd/buildah/run.go

```diff
@@ -84,19 +71,17 @@ func runCmd(c *cli.Context) error {
 
 	options := buildah.RunOptions{
 		Hostname: c.String("hostname"),
-		Runtime:  runtime,
-		Args:     flags,
+		Runtime:  c.String("runtime"),
+		Args:     c.StringSlice("runtime-flag"),
 	}
 
-	if c.IsSet("tty") {
-		if c.Bool("tty") {
-			options.Terminal = buildah.WithTerminal
-		} else {
-			options.Terminal = buildah.WithoutTerminal
-		}
+	if c.Bool("tty") {
+		options.Terminal = buildah.WithTerminal
+	} else {
+		options.Terminal = buildah.WithoutTerminal
 	}
 
-	for _, volumeSpec := range volumes {
+	for _, volumeSpec := range c.StringSlice("volume") {
```

Comment:
I'm going back/forth.  Should "volume" be "volumes"?  Just seems odd to have a range against something that's singular.  We are dealing with two volumes, but one string, so I guess you could make a case for either?  I'll stop thinking.

Reply:
I think we should leave this.

Reply:
👍